app/models/manageiq/providers/openstack/infra_discovery.rb

```diff
@@ -0,0 +1,17 @@
+require 'manageiq/network_discovery/port_scanner'
+
+module ManageIQ::Providers::Openstack
+  class InfraDiscovery
+    def self.probe(ost)
+      res = ""
+      if PortScanner.portOpen(ost, 6385)
```

Comment:
If possible, can we change the PortScanner to be fully namespaced under ManageIQ::NetworkDiscovery, since you are already making changes in there?

Reply:
Absolutely! Thanks